Frontend/android-native/app/src/main/java/in/juspay/mobility/MainActivity.java

```diff
@@ -724,6 +735,14 @@ protected void onDestroy() {
         if (sharedPref != null) {
             sharedPref.edit().putString(getResources().getString(in.juspay.mobility.app.R.string.ACTIVITY_STATUS), "onDestroy").apply();
         }
+        if (triggeredFinishAffinity)
+        {
+            triggeredFinishAffinity = false;
+            handler.removeCallbacks(runnable);
```

Comment:
make handler ,runnable = null after calling removeCallbacks. and add a null check also